splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/AbstractDropIndexConstantOperation.java

```diff
@@ -126,6 +123,7 @@ private void drop(ConglomerateDescriptor cd,
         SpliceTransactionManager userTxnManager = (SpliceTransactionManager)lcc.getTransactionExecute();
         dd.dropConglomerateDescriptor(cd,userTxnManager);
         td.removeConglomerateDescriptor(cd);
+        dd.deletePartitionStatistics(cd.getConglomerateNumber(), userTxnManager);
```

Comment:
We may need to broadcast the ALTER_STATS messages to invalidate the plans based on that stats too and remove the entry from the dictionary cache. See StatisticsAdmin.ddlNotification().

Reply:
Good to know! Added in the next commit.